src/DevTools/setupTestWrapper.tsx

```diff
@@ -86,12 +86,12 @@ export const setupTestWrapperTL = <T extends OperationType>({
   const renderWithRelay = (
     mockResolvers: MockResolvers = {},
     manualEnvControl?: boolean,
-    componentProps?: {}
+    componentProps?: {},
+    mockedEnv: MockEnvironment = createMockEnvironment()
   ): RenderWithRelay => {
-    const env = createMockEnvironment()
     const TestRenderer = () => (
       <QueryRenderer<T>
-        environment={env}
+        environment={mockedEnv}
```

Comment:
After some wrestling I have got the whole form working in tests - it required a small adjustment to our rendering harness. Interested in feedback on this approach, as after implementing it I realized it looks similar to how our enzyme-based `renderWithRelay` did it.

Reply:
we should keep things consistent with how we do it below, or update the below so that it is consistent with whats here

Reply:
Do you mean match [this format](https://github.com/artsy/force/blob/f043d0db90852bca50854df7fabf98b871ee77f4/src/DevTools/setupTestWrapper.tsx#L129-L139) for falling back to creating the env, or matching all the positional arguments in both? Because they are positional i think this could be a huge diff on an already big PR.